homeassistant/components/tuya/config_flow.py

```diff
@@ -389,6 +393,14 @@ def _get_climate_schema(curr_conf, device):
                     CONF_CURR_TEMP_DIVIDER,
                     default=curr_conf.get(CONF_CURR_TEMP_DIVIDER, 0),
                 ): vol.All(vol.Coerce(int), vol.Clamp(min=0)),
+                vol.Optional(
+                    CONF_SET_TEMP_DIVIDED,
+                    default=curr_conf.get(CONF_SET_TEMP_DIVIDED, True),
```

Comment:
If the default is true, do we need to migrate the config entry to version 2 so all the existing users get the change?

Reply:
What does this option mean?

Reply:
Because the temperature provided by the device could be divided inside the library by the value set by the previous options `TEMP_DIVIDER`, this parameter instruct the library in the `set_temperature` method if to use this divider also for the set command. A parameter is required because different devices have different behavior and there is no way to determinate automatically the correct one. Using value `True` or `False` as default is just a choice, `True` seems more logic because is using in the set temperature command the same value returned as target temperature.
Default = True is also managed in the climate class, so I don't think is required to migrate config entry to a new version. If option is not available is set to `True`.

Reply:
I explained with an example here to be more clear.

Suppose that the device report for target temperature  the value 210. 
Setting `divider` to 10 in the library, target temperature returned by tuyaha will became 21.0.

If now I send a `set_temperature` command to the library with value 22.0, I have 2 option:
- case 1, `use_divider = true` -> `set_temperature` will send to the device value 22.0
- case 2, `use_divider = false` -> `set_temperature` will send to the device value 220

Reply:
Please add a test to the options flow for the new options

Reply:
Quite complex, I should mock Tuya class, you know that this option flow allow go through device properties based on devices type... why this is required now?

Reply:
All config flows are required to have 100% coverage.  It should have been required before.

Reply:
Ah, ok. As I said is a little bit complex for this flow, I'll work on this. In this case I have also to correct `.coveragerc`,  currently is excluding everything for tuya (*).